src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -419,6 +423,11 @@ impl Coordinator {
         resolved_ids: ResolvedIds,
     ) -> Result<ExecuteResponse, AdapterError> {
         let connection_oid = self.catalog_mut().allocate_oid()?;
+        let cloud_resource_controller = self
+            .cloud_resource_controller
+            .as_ref()
+            .cloned()
+            .ok_or(AdapterError::Unsupported("AWS PrivateLink connections"))?;
```

Comment:
Where did this come from? Or am I missing something obvious?

Reply:
This was split out from an error case which used to occur _after_ the catalog transaction. But I moved this up so we'd error before the transaction. Turns out this made a bunch of tests fail, so I refactored it again to just emit warnings